{"uuid": "16f41db3-a12b-4089-9b8c-9d54e83efb45", "children": ["2974798f-94d4-4d7e-a578-febfcb117b56"], "befores": [{"name": "create_section", "status": "broken", "statusDetails": {"message": "KeyError: 'id'\n", "trace": "  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 910, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/todo_api/conftest.py\", line 47, in create_section\n    id_section_created = section_created[\"body\"][\"id\"]\n                         ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^\n"}, "start": 1712607416639, "stop": 1712607416958}], "start": 1712607416639, "stop": 1712607416980}